src/h264/H26xPacketizer.cpp

```diff
@@ -54,22 +53,22 @@ void H26xPacketizer::EmitNal(VideoFrame& frame, BufferReader nal, std::string& f
 	//Skip payload nal header
 	nal.Skip(naluHeaderSize);
 	pos += naluHeaderSize;
-	//Split it
-	while (nal.GetLeft())
+	uint32_t numPackets = std::ceil(static_cast<double>(nal.GetLeft()) / (RTPPAYLOADSIZE - fuPrefixSize));
```

Comment:
Looks if mod != 0, one  more bytes will be added to each packet. Should we change this to?
```suggestion
	uint32_t numPackets = std::ceil(static_cast<double>(nal.GetLeft()) / (RTPPAYLOADSIZE - fuPrefixSize - 1));
```

Reply:
OK. Probably not an issue as you calculate packet len later.